include/pmtf/wrap.hpp

```diff
@@ -20,10 +20,15 @@ namespace pmtf {
 inline std::ostream& operator<<(std::ostream& os, const map& value);
 bool operator==(const pmt& x, const map& y);
 inline bool operator==(const map& y, const pmt& x) { return operator==(x,y); }
-std::ostream& operator<<(std::ostream& os, const pmt& value);
```

Comment:
What is the general purpose of `wrap` at this point since most everything that was wrap got moved into the base class